backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -274,14 +274,14 @@ public static Boolean parseYesNo(String v) {
   }
 
   // "NA" is used for international addresses
-  private static final Set<String> STATE_CODES =
+  public static final Set<String> STATE_CODES =
       Set.of(
           "AK", "AL", "AR", "AS", "AZ", "CA", "CO", "CT", "DC", "DE", "FL", "FM", "GA", "GU", "HI",
           "IA", "ID", "IL", "IN", "KS", "KY", "LA", "MA", "MD", "ME", "MH", "MI", "MN", "MO", "MP",
           "MS", "MT", "NC", "ND", "NE", "NH", "NJ", "NM", "NV", "NY", "OH", "OK", "OR", "PA", "PR",
           "PW", "RI", "SC", "SD", "TN", "TX", "UT", "VA", "VI", "VT", "WA", "WI", "WV", "WY", "NA");
 
-  private static final Set<String> CANADIAN_STATE_CODES =
+  public static final Set<String> CANADIAN_STATE_CODES =
```

Comment:
non-blocking thought and not necessarily something we need to solve right now: how did you know to find these here? I wonder how many different places in the code we have defined the different states, different org types, different genders, etc. and if it makes sense to consolidate them in some way

Reply:
Translators should be the main place for US/Canadian states in the backend, I don't think we have them anywhere, though I might be wrong, as far as org types i am not sure. For genders, I included that because the values that I stated come from the RS docs and don't have to extract match to the rest of the app. I see what you're saying though, maybe its worth creating a ticket for all the areas you know that needs to be consolidated and we can keep adding things to it until it has enough things to be addressed